zcash_primitives/src/transaction/components/sapling.rs

```diff
@@ -5,7 +5,8 @@ use group::GroupEncoding;
 use std::io::{self, Read, Write};
 
 use zcash_note_encryption::{
-    EphemeralKeyBytes, ShieldedOutput, COMPACT_NOTE_SIZE, ENC_CIPHERTEXT_SIZE,
+    EphemeralKeyBytes, KeyedOutput, RecoverableOutput, ShieldedOutput, COMPACT_NOTE_SIZE,
+    ENC_CIPHERTEXT_SIZE, OUT_CIPHERTEXT_SIZE,
 };
```

Comment:
Fix the clippy warning below by deriving `Eq`.

Reply:
Fixed in #645